tools/operator/README.md

```diff
@@ -0,0 +1,107 @@
+- Fabric binaries in $PATH <https://hyperledger-fabric.readthedocs.io/en/latest/install.html>
+
+#### Kubernetes
+
+- Kubernetes cluster if launching fabric network on kubernetes cluster. It is the responsibility
+of the user to carefully consider and to create a kube cluster with enough resources to handle
+the number of nodes specified in networkspec for the planned traffic patterns and rates
+- `kubectl` if launching fabric network on kubernetes cluster. Refer to
+<https://kubernetes.io/docs/tasks/tools/install-kubectl/> for installing `kubectl`
+
+#### Local using docker
+
+- Docker <https://docs.docker.com/install/linux/docker-ce/ubuntu/> and Docker-compose
+<https://docs.docker.com/compose/install/> if launching fabric network locally
+
+## Usage
+
+Supported input arguments for launcher are:
+
+```go
+-a string
+       Set action(up, down, create, join, anchorpeer, install, instantiate, upgrade, 
+	   invoke, query, createChannelTxn, migrate, health) (default is up)
+-i string
+       Network spec input file path (Required)
+-k string
+       Kube config file path (If omitted, then use local network)
+```
+
+- `-a` is used to set type of action to be performed. It takes all the above actions as the values. Default value is up.
+#####Actions that uses network input file
+		`up`		 			  To launch a fabric network.
+		`down`				   To take down a network
+		`createChannelTxn`To create channelTx for specified number of channels
+		`migrate` 			   To migrate a network to etcdraft
+		`health`				   To perform health check on peers and orderers
+		`upgradeNetwork`   To upgrade an existing fabric network
+#####Actions that uses test input file
+		`create`			      To create a channel
+		`join`					  To join peers to a channel
+		`anchorpeer`		  To perform anchor peer update
+		`install`				   To install a chaincode
+		`instantiate`		    To instantiate a chaincode
+		`upgrade`			   To upgrade a chaincode
+		`invoke`				  To perform invokes by sending the traffic to a fabric network
+		`query`				   To perform queries on a fabric network
+
+- `-i` is used to pass the absolute file path for network input file. It is required
+to launch/remove fabric network. Network input file can be prepared using `networkInput.md`
+in operator
+- `-k` is used to pass the absolute file path to kube config file of kubernetes cluster.
+If `-k` is specified with `path/to/kube-config.yaml`, then it will launch/remove
+fabric network on kubernetes cluster. Kube config file can be downloaded from kubernetes
+cluster. If `-k` is not specified in the command line, launcher will launch fabric
+network locally using docker-compose
+
+### Examples
+
+#### On Kubernetes Cluster
+
+To launch fabric network in kubernetes cluster, need kube config file for cluster
+and network input file
+```go run launcher.go -i <path/to/network spec file> -k <path/to/kube config file> -a up``` or
+```go run launcher.go -i <path/to/network spec file> -k <path/to/kube config file>```
```

Comment:
I think we have to run these commands from the same directory. It's better to mention from which directory we have to execute these commands.

Reply:
Forgot to change launcher to main. We will be running everything from the operator directory.